src/js/auth.js

```diff
@@ -46,10 +48,16 @@ function login(success, config) {
       return;
     }
 
-    auth0.loginWithRedirect({
-      appState: location.pathname,
-      prompt: 'none',
-    });
+    if (location.pathname === '/login') {
+      auth0.loginWithRedirect({
+        appState: '/',
```

Comment:
actually this is breaking..  we need to be able to pass the appState from the original url.. and this'll make users login each time.  the card instructions were wrong.  hmm we also don't want to go to the login screen at all unless it's needed...